dumpling/export/config.go

```diff
@@ -221,6 +222,13 @@ func (conf *Config) GetDriverConfig(db string) *mysql.Config {
 	driverCfg.MaxAllowedPacket = 0
 	if conf.Security.DriveTLSName != "" {
```

Comment:
Seems we also don't need it now, originally we use `DriveTLSName` for `RegisterTLSConfig`. Now we can remove `registerTLSConfig` and change `DriveTLSName` to `TLSConfig` to let `driverCfg.TLS` use it.

And can you also do this adjustment for lightning in next PR?